metrics/sinks/hawkular/client.go

```diff
@@ -124,6 +136,20 @@ func (self *hawkularSink) idName(ms *core.MetricSet, metricName string) string {
 	return strings.Join(n, separator)
 }
 
+func (self *hawkularSink) nodeName(ms *core.MetricSet) string {
+	if &self.labelNodeId != nil {
```

Comment:
I updated this to remove one of the else statements in there, not sure if that was exactly what you were expecting or not though.